components/clarinet-deno/index.ts

```diff
@@ -641,17 +658,17 @@ String.prototype.expectTuple = function () {
       stack.pop();
     }
   }
-  let remainder = this.substring(start, this.length - 1);
+  const remainder = this.substring(start, this.length - 1);
   if (remainder.length > 0) {
     elements.push(remainder);
   }
 
-  let tuple: Record<string, string> = {};
-  for (let element of elements) {
-    for (var i = 0; i < element.length; i++) {
+  const tuple: Record<string, string> = {};
+  for (const element of elements) {
+    for (let i = 0; i < element.length; i++) {
       if (element.charAt(i) === ":") {
-        let key: string = element.substring(0, i);
-        let value: string = element.substring(i + 2, element.length);
+        const key = element.substring(0, i).trim();
+        const value = element.substring(i + 2).trim();
```

Comment:
The `trim`s here could break existing tests (in some rare cases), but probably for the better ;

`expectTuple("{ id: 1 }")` used to return `{ " id": "1 "}`

Reply:
ah indeed, thanks!